test/commands/scanner/rule/remove.test.ts

```diff
@@ -37,154 +30,108 @@ const customPathDescriptor = {
+				const output = runCommand(`scanner rule remove`);
+				const expectedRuleSummary = [pathToApexJar1, pathToApexJar2, pathToApexJar3]
+					.map(p => messages.getMessage('output.dryRunRuleTemplate', [p]))
+					.join('\n');
+				expect(output.shellOutput.stdout).to.contain(messages.getMessage('output.dryRunOutput', [3, expectedRuleSummary]), 'All paths should be logged');
+			});
 
-		describe('Rule Removal', () => {
-			describe('Test Case: Removing a single PMD JAR', () => {
-				removeTest
-					// We'll wait a few seconds then send in a 'y', to simulate the user confirming the request.
-					.stdin('y\n', waitTime)
-					.timeout(10000)
-					.command(['scanner:rule:remove',
-						'--path', pathToApexJar1
-					])
-					.it('The specified JAR is deleted.', ctx => {
-						expect(ctx.stdout).to.contain(
-							messages.getMessage('output.resultSummary', [pathToApexJar1]),
-							'Console should report deletion.'
-						);
-						const updatedCustomPathJson = getCustomPathFileContent();
-						expect(updatedCustomPathJson).to.deep.equal({
-							'pmd': {
-								'apex': [pathToApexJar2, pathToApexJar3]
-							}
-						}, `Deletion should have been persisted ${JSON.stringify(updatedCustomPathJson)}`);
-					});
+			// TODO: THIS TEST CANNOT BE ENABLED UNTIL WE SWITCH FROM `SfdxCommand` TO `SfCommand`!
+			xit('By default, rule removal must be confirmed', async () => {
+				const output = await runInteractiveCommand(`scanner rule remove --path ${pathToApexJar1}`, {
+					'These rules will be unregistered': Interaction.Yes
+				});
+				expect(output.stdout).to.contain(
+					messages.getMessage('output.resultSummary', [pathToApexJar1]),
+					'Console should report deletion.'
+				);
+				const updatedCustomPathJson = getCustomPathFileContent();
+				expect(updatedCustomPathJson).to.deep.equal({
+					'pmd': {
+						'apex': [pathToApexJar2, pathToApexJar3]
+					}
+				}, `Deletion should have been persisted ${JSON.stringify(updatedCustomPathJson)}`);
 			});
 
-			describe('Test Case: Removing multiple PMD JARs', () => {
-				removeTest
-					// We'll wait a few seconds then send in a 'y', to simulate the user confirming the request.
-					.stdin('y\n', waitTime)
-					.timeout(10000)
-					.command(['scanner:rule:remove',
-						'--path', [pathToApexJar1, pathToApexJar2].join(',')
-					])
-					.it('The specified JARs are deleted', ctx => {
-						expect(ctx.stdout).to.contain(
-							messages.getMessage('output.resultSummary', [[pathToApexJar1, pathToApexJar2].join(', ')]),
-							'Console should report deletion'
-						);
-						const updatedCustomPathJson = getCustomPathFileContent();
-						expect(updatedCustomPathJson).to.deep.equal({
-							'pmd': {
-								'apex': [pathToApexJar3]
-							}
-						}, 'Deletion should have been persisted');
-					});
+			// TODO: THIS TEST CANNOT BE ENABLED UNTIL WE SWITCH FROM `SfdxCommand` TO `SfCommand`!
+			xit('Rule removal can be safely aborted', async () => {
```

Comment:
This test stays disabled until we move to `SfCommand`, since the test interactivity won't work properly until then.